solutions/usaco-831.mdx

```diff
@@ -2,13 +2,125 @@
 id: usaco-831
 source: USACO Bronze 2018 Open
 title: Team Tic Tac Toe
-author: Jesse Choe, Kevin Sheng, Benjamin Qi
+author: Jesse Choe, Kevin Sheng, Benjamin Qi, Brad Ma
 ---
 
 ## Solution 1
 
 [Official Analysis](http://www.usaco.org/current/data/sol_tttt_bronze_open18.html)
 
+<LanguageSection>
+	
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class TeamTicTacToe {
+	static char[][] Board = new char[3][3];
+
+	// Does 1 cow win?
+	static int cowWins(char ch) {
+		// Check diagonals
+		if (Board[0][0] == ch && Board[1][1] == ch && Board[2][2] == ch) {
+			return 1;
+		}
+		if (Board[0][2] == ch && Board[1][1] == ch && Board[2][0] == ch) {
+			return 1;
+		}
```

Comment:
Totally up to you, but could you change this into a loop?

Reply:
given that there's only 3 elements it shouldn't matter much
also why is it still named `Board`